CONTRIBUTORS.md

```diff
@@ -77,7 +77,8 @@ submit the change with your pull request.
 - Jim Stallings / [jestallin](https://github.com/jestallin)
 - Alyssa Ransbury / [alran](https://github.com/alran)
 - Thomas Countz / [thomascountz](https://github.com/thomascountz)
+- Megan Talbot / [meganft](https://github.com/meganft)
 
 ## Bots
 
-- Security and Dependency Updates / [deppbot](https://github.com/deppbot)
\ No newline at end of file
+- Security and Dependency Updates / [deppbot](https://github.com/deppbot)
```

Comment:
i used the github web editor for this PR -- so it has modified that last line but i can't see what it did.